.github/sync-repo-settings.yaml

```diff
@@ -46,4 +47,4 @@ permissionRules:
 - team: yoshi-java-admins
   permission: admin
 - team: yoshi-java
-  permission: push
\ No newline at end of file
+  permission: push
```

Comment:
are these config changes necessary for this PR? If no, it might be better to split them up

Reply:
I'm actually not sure how this got added, let me remove the changes in this file.